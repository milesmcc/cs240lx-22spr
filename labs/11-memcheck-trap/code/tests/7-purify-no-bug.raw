opened tty port </dev/cu.usbserial-0001>.
my-install: tty-usb=</dev/cu.usbserial-0001> program=<./tests/7-purify-no-bug.bin> about to boot
TRACE:simple_boot: sending 29960 bytes, crc32=9528c0e
waiting for a start...
PRINT_STRING:pi sent print: <glen3b: success: Received the program!>
bootloader: Done.
TRACE:should have no error: running w verbose off
TRACE:ERROR:Allocated block 1 [0x183ef8] corrupted at offset -4
TRACE:	logical block id=1, [addr=0x183ef8] nbytes=4
TRACE:	Block allocated at: tests/7-purify-no-bug.c:fn:9
TRACE:ERROR:Allocated block 1 [0x183ef8] corrupted at offset -3
TRACE:	logical block id=1, [addr=0x183ef8] nbytes=4
TRACE:	Block allocated at: tests/7-purify-no-bug.c:fn:9
PANIC:../../labs/9-memcheck-stat/code.staff/ckalloc.c:ckalloc:150:mem_check(h, ck_get_rz1(h), rn)
DONE!!!

bootloader: pi exited.  cleaning up
