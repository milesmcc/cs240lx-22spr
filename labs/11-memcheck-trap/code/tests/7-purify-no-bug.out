TRACE:should have no error: running w verbose off
TRACE:ERROR:Allocated block 1  corrupted at offset -4
TRACE:	logical block id=1,  nbytes=4
TRACE:	Block allocated at: tests/7-purify-no-bug.c:fn:9
TRACE:ERROR:Allocated block 1  corrupted at offset -3
TRACE:	logical block id=1,  nbytes=4
TRACE:	Block allocated at: tests/7-purify-no-bug.c:fn:9
PANIC:../../labs/9-memcheck-stat/code.staff/ckalloc.c:ckalloc:150:mem_check(h, ck_get_rz1(h), rn)
